lib/puppet/provider/exec/pwsh.rb

```diff
@@ -67,4 +79,10 @@ def native_path(path)
   def legacy_args
     '-NoProfile -NonInteractive -NoLogo -ExecutionPolicy Bypass'
   end
+
+  def posix_args
+    # Note - using -ExecutionPolicy causes PowerShell to abort
```

Comment:
Is this still an issue? The linked GitHub issue is closed as fixed in 6 Alpha 18.

Reply:
Confirmed not an issue.  will remove.

Reply:
Fixed